include/deal.II/cgal/triangulation.h

```diff
@@ -164,6 +198,102 @@ namespace CGALWrappers
 
 
 
+  template <typename CGALTriangulationType,
+            typename CornerIndexType,
+            typename CurveIndexType>
+  void
+  cgal_triangulation_to_dealii_triangulation(
+    const CGAL::Mesh_complex_3_in_triangulation_3<CGALTriangulationType,
+                                                  CornerIndexType,
+                                                  CurveIndexType>
+      &               cgal_triangulation,
+    Triangulation<3> &dealii_triangulation)
+  {
+    using C3T3 = CGAL::Mesh_complex_3_in_triangulation_3<CGALTriangulationType,
+                                                         CornerIndexType,
+                                                         CurveIndexType>;
+
+    // Extract all vertices first
+    std::vector<Point<3>> dealii_vertices;
+    std::map<typename C3T3::Vertex_handle, unsigned int>
+      cgal_to_dealii_vertex_map;
+
+    std::size_t inum = 0;
+    for (auto vit = cgal_triangulation.triangulation().finite_vertices_begin();
+         vit != cgal_triangulation.triangulation().finite_vertices_end();
+         ++vit)
+      {
+        if (vit->in_dimension() <= -1)
+          continue;
+        cgal_to_dealii_vertex_map[vit] = inum++;
+        dealii_vertices.emplace_back(
+          CGALWrappers::cgal_point_to_dealii_point<3>(vit->point()));
+      }
+
+    // Now build cell connectivity
+    std::vector<CellData<3>> cells;
+    for (auto cgal_cell = cgal_triangulation.cells_in_complex_begin();
+         cgal_cell != cgal_triangulation.cells_in_complex_end();
+         ++cgal_cell)
+      {
+        CellData<3> cell(4);
```

Comment:
Could we use `ReferenceCell::n_vertices()` here and below? Even if this only works with tets I would prefer not to hardcode these values.

Reply:
@drwells, done. 👍